.github/workflows/release.yml

```diff
@@ -29,6 +29,14 @@ jobs:
           plutil -replace CFBundleVersion -string ${{ github.event.inputs.version }} -- 'Demo/Application/Supporting Files/Braintree-Demo-Info.plist'
           plutil -replace CFBundleVersion -string ${{ github.event.inputs.version }} -- 'Sources/BraintreeCore/Info.plist'
 
+      # Remove the SPMTest app to prevent Carthage timeout; store SPMTest app in temp dir
+      - name: Remove SPMTest demo app
+        run: |
+          echo "TEMP_DIR=$(mktemp -d)" >> $GITHUB_ENV
+          mv SampleApps/SPMTest/ ${{ env.TEMP_DIR }}/
+          git add SampleApps/SPMTest
+          git commit -m 'Remove SPMTest app to avoid Carthage timeout'
```

Comment:
Would it make sense for the commit message to indicate that this is part of the release process? Maybe it could be:

```
Pre-release: Remove SPMTest app to avoid Carthage timeout
```

Reply:
DONE